scipy/stats/_distn_infrastructure.py

```diff
@@ -15,10 +15,10 @@
 
 from scipy.misc import doccer
 from ._distr_params import distcont, distdiscrete
-from scipy._lib._util import check_random_state, _lazywhere
+from scipy._lib._util import check_random_state
```

Comment:
I think this is not right: `_lazywhere` was recently moved to `_lib._util`: https://github.com/scipy/scipy/blob/master/scipy/_lib/_util.py#L26

This might be the cause of the CI failure.